.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,4 +15,6 @@ Anything you think will be useful for reviewers.
 These aren't hard requirements, just guidelines
 
 - [ ] New/modified Rust code formatted with `cargo fmt`
+- [ ] Documentation and `README.md` updated for this change
```

Comment:
should probably qualify as "if necessary", just so people don't feel they have to update it (in the case of e.g. refactors etc.).

Reply:
I've added that in the latest commit, thanks